jobs/config.json

```diff
@@ -11231,6 +11231,22 @@
       "sig-scalability"
     ]
   },
+  "pull-kubernetes-local-e2e": {
+    "args": [
+      "--deployment=local",
+      "--extract=ci/latest",
+      "--extract-source",
```

Comment:
you should use the source from the PR, rather than ci/latest

Reply:
aka, in presubmit, you already have the full source of the PR

Reply:
ack removing both `--extract` and `--extract-source`

Reply:
looks like i need at least `--extract=local`

Reply:
@krzyzacy is `--extract=local` the right param?

Reply:
do you expect to use the built tarball or not?

Reply:
If so you need --build and --stage then, if not you shouldn't need to extract anything